### PR TITLE
ROSA support topic.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
This topic is specifically for ROSA because customers can contact either Red Hat or AWS for support.

QE and peer review needed.

JIRA: https://issues.redhat.com/browse/OSDOCS-1779

### CODE REVIEW

**Comment by jonquilwilliams** [comment]
@yuwang-RH  @vikram-redhat  @jeremyeder  @okashi18  @davemulford

File: modules/rosa-getting-support.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* Submit a support case to Red Hat Support:
+.. Click *Open a New Case*.
+.. Select the reason for the support ticket, such as `Defect/Bug` or `Account/Customer Service Request`.
+.. In the `Product` field, enter `OpenShift` to filter the list. Select `{product-title}` and the version from the drop-down menus.
+.. Complete the remaining fields.
+.. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
+
+You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support].
+
+If you have a suggestion for improving this documentation or have found an error, submit a link:https://bugzilla.redhat.com/index.cgi[Bugzilla] report against the OpenShift Container Platform product for the Documentation component. Please provide specific details, such as the section name and {product-title} version.
```
Comment (bobfuru) [inline modules/rosa-getting-support.adoc]: We generally avoid "please" in our docs. I think it's a minor thing in this case because it's directing the customer to submit a bug, which seems like a notable exception. But you could also say, "Be sure to provide..." ¯\_(ツ)_/¯

**Review by bobfuru — COMMENTED** [review]
One possible suggestion and otherwise LGTM!

File: modules/rosa-getting-support.adoc (hunk 1 above)
Comment (jonquilwilliams) [reply modules/rosa-getting-support.adoc]: Good catch, thanks!

**Comment by davemulford** [comment]
This looks good to me!

File: modules/rosa-getting-support.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Submit a support case to Red Hat Support:
+.. Click *Open a New Case*.
+.. Select the reason for the support ticket, such as `Defect/Bug` or `Account/Customer Service Request`.
+.. In the `Product` field, enter `OpenShift` to filter the list. Select `{product-title}` and the version from the drop-down menus.
+.. Complete the remaining fields.
+.. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
+
+You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support].
+
+If you have a suggestion for improving this documentation or have found an error, submit a link:https://bugzilla.redhat.com/index.cgi[Bugzilla] report against the OpenShift Container Platform product for the Documentation component. Be sure to provide specific details, such as the section name and {product-title} version.
```
Comment (jeremyeder) [inline modules/rosa-getting-support.adoc:25]: Is that really where you want customers to file issues against managed services docs?  Directly in BZ?  Shouldn't it go through the same flow described earlier in this PR (RH customer portal or AWS support).

File: modules/rosa-getting-support.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* Search or browse through the Red Hat Knowledgebase of technical support articles about Red Hat products.
+* Access other product documentation.
+* Submit a support case to Red Hat Support:
+.. Click *Open a New Case*.
+.. Select the reason for the support ticket, such as `Defect/Bug` or `Account/Customer Service Request`.
+.. In the `Product` field, enter `OpenShift` to filter the list. Select `{product-title}` and the version from the drop-down menus.
+.. Complete the remaining fields.
+.. On the _Review_ page, select the correct cluster ID that you are contacting support about, and click `Submit`.
+
+You can also get support from link:https://aws.amazon.com/premiumsupport/[AWS Support].
```
Comment (jeremyeder) [inline modules/rosa-getting-support.adoc:23]: @davemulford  @jgouldin you ok with this language and doc?

File: modules/rosa-getting-support.adoc (hunk 2 above)
Comment (davemulford) [reply modules/rosa-getting-support.adoc:25]: Paying customers are normally funneled through support, but there is nothing stopping them from taking any other path.  However, if it has caused them some issue there is little way to prioritize that back to an incident if they created a BZ directly.

TL;DR is that it definitely benefits the customer to open a support case, if a piece of the docs is causing them any issue.

Comment (jeremyeder) [reply modules/rosa-getting-support.adoc:25]: I don't want customers to ever go to BZ.  I don't think we should document anything other than "call support".

Comment (jonquilwilliams) [reply modules/rosa-getting-support.adoc:25]: @jeremyeder  @davemulford  This Bugzilla info mirrors the OSD info. Just FYI. It's your call. LMK

File: modules/rosa-getting-support.adoc (hunk 3 above)
Comment (davemulford) [reply modules/rosa-getting-support.adoc:23]: I'm okay with this.

File: modules/rosa-getting-support.adoc (hunk 2 above)
Comment (davemulford) [reply modules/rosa-getting-support.adoc:25]: I'm okay with directing customers to open a support case. It helps with prioritization.
